packages/react-output-target/react-component-lib/utils/attachProps.ts

```diff
@@ -62,6 +62,17 @@ export const getClassName = (classList: DOMTokenList, newProps: any, oldProps: a
   return finalClassNames.join(' ');
 };
 
+/**
+ * Transforms a React event name to a browser event name.
+ */
+export const transformReactEventName = (eventNameSuffix: string) => {
+  switch (eventNameSuffix) {
+    case 'doubleclick':
+      return 'dblclick';
```

Comment:
The `doubleclick` vs `dblclick` naming is a React thing correct? So React devs need to use `onDoubleClick`?

Reply:
Correct, React developers would use `onDoubleClick`. This is an odd scenario where the JSX casing doesn't match the "standard" practice of aligning with the browser event name (e.g. `onClick` → `click`, `onInput` → `input`).

Reply:
Thanks!